src/Demo.vue

```diff
@@ -66,13 +66,8 @@
 </template>
```

Comment:
We still need the import coming from here since `demo.js` (I believe) will not help in cypress land.

```js
<script setup>
  import { A11yDialog } from '../dist/vue-a11y-dialog.esm.js'
</script>
<script>
  export default {
...and so on

```